docs/contrib/contributing.rst

```diff
@@ -3,26 +3,27 @@
 Contributing guidelines
 =======================
 
-Very important notice about the Javalib
----------------------------------------
+Very important notice about Javalib
+-----------------------------------
 
-Scala Native contains a reimplementation of part of the JDK in Scala.
+Scala Native contains a re-implementation of part of Java.
```

Comment:
Saying "Java" is probably a much graver violation of the terms of the Java trademark. Oracle is very picky about what things can be called "Java". Scala Native and Scala.js are not "Java", and do not reimplement "Java". They reimplement a part of the JDK API.